README.md

```diff
@@ -81,8 +81,8 @@ var pact = require('@pact-foundation/pact-node');
 
 pact.verifyPacts({
 	providerBaseUrl: <String>,           // Running API provider host endpoint. Required.
-	pactBrokerUrl: <String>              // URL of the Pact Broker to retrieve pacts from. Required if not using pactUrls.
-	provider: <String>                   // Name of the Provider. Required if not using pactUrls.
+	pactBrokerUrl: <String>              pactFilesOrDirs
```

Comment:
Any reason why we removed the human readable description?

Reply:
You mean the comments?  I just feel it's a bit redundant as we already have them in the readme.